content/docs/guides/parallel-execution.md

```diff
@@ -0,0 +1,396 @@
+ 
+Below is the basic command to start the execution.
+```shell
+java -cp <classpath> cucumber.api.cli.Main -g <steps package> --threads <thread count> <path to feature files>
+```
+
+Follow the below steps to **execute the command from a terminal**.
+
+- Download the necessary cucumber jars from the cucumber.io Maven repository (links provided under version number) or, if available, copy from the local repository into a folder. These need to be provided in the **classpath**. The versions mentioned are compatible with the latest cucumber release 4.3.0.
+
+	1.  `cucumber-core.jar` [`4.3.0`](http://central.maven.org/maven2/io/cucumber/cucumber-core/4.3.0/cucumber-core-4.3.0.jar)
+	2.  `cucumber-java.jar` [`4.3.0`](http://central.maven.org/maven2/io/cucumber/cucumber-java/4.3.0/cucumber-java-4.3.0.jar)
+	3.  `cucumber-java8.jar` [`4.3.0`](http://central.maven.org/maven2/io/cucumber/cucumber-java8/4.3.0/cucumber-java8-4.3.0.jar)
+	4.  `cucumber-html.jar` [`0.2.7`](http://central.maven.org/maven2/io/cucumber/cucumber-html/0.2.7/cucumber-html-0.2.7.jar)
+	5.  `gherkin.jar` [`5.1.0`](http://central.maven.org/maven2/io/cucumber/gherkin/5.1.0/gherkin-5.1.0.jar)
+	6.  `cucumber-expressions.jar` [`6.2.2`](http://central.maven.org/maven2/io/cucumber/cucumber-expressions/6.6.2/cucumber-expressions-6.6.2.jar)
+	7.  `datatable.jar` [`1.1.12`](http://central.maven.org/maven2/io/cucumber/datatable/1.1.12/datatable-1.1.12.jar)
+	8.  `datatable-dependencies.jar` [`1.1.12`](http://central.maven.org/maven2/io/cucumber/datatable-dependencies/1.1.12/datatable-dependencies-1.1.12.jar)
+	9.  `tag-expressions.jar` [`1.1.1`](http://central.maven.org/maven2/io/cucumber/tag-expressions/1.1.1/tag-expressions-1.1.1.jar)
+	10. `typetools.jar` [`0.5.0`](http://central.maven.org/maven2/net/jodah/typetools/0.5.0/typetools-0.5.0.jar)
+	
+- Create a folder named `cukecli` and a folder `parallel` inside it, to keep the step definition classes and feature files.
+
+- Create the **two feature files** (`scenarios.feature` and `scenariooutlines.feature`) inside the `parallel` folder as described in the JUnit section.
+
+- Create the **step definition class** in the `parallel` folder as described in the JUnit section.
+
+- Open up a **terminal window** and navigate to the source folder of the project, in our case `cukecli`.
+
+{{% block "java" %}}
+- Compile the step definition class. Add the **path to the folder containing cucumber jars to the classpath** using the **-cp** option.
+
+```shell
+javac -cp .;<path to cucumber jar folder>/* ./parallel/Stepdefs.java
+```
+{{% /block %}}
+
+{{% block "kotlin" %}}
+- Compile the step definition class. Add **path to each of the downloaded cucumber jars to the classpath** using the **-cp** option.
+
+```shell
+kotlinc -cp .;<path to each cucumber jar> -jvm-target 1.8 ./parallel/Stepdefs.java
+```
+{{% /block %}}
+
+- Execute using the below command.
+
+{{% block "java" %}}

```

Comment:
Same here.
Note that anything **outside** the language blocks would be visible on all polyglot pages (if we add javascript/ruby in the future).

Reply:
Will change this